guide-us-english.tex

```diff
@@ -164,26 +163,27 @@ \subsection{Values-Driven Culture}
 this must occur in an environment based on respect for the individuals
 doing the work, without whom nothing can be created.
 
-Scrum@Scale helps organizations thrive by supporting both a
-servant-leadership and intent-based leadership model,\footnote{Marquet, L
-David, Turn the Ship Around!: How to Create Leadership at Every Level,
-Greenleaf Book Group, 2012} which fosters a positive environment for
+Scrum@Scale helps organizations thrive by supporting a
+transformational leadership model which fosters a positive environment for
 working at a sustainable pace and putting commitment to deliver
 customer-facing value at the forefront of our efforts.
 
 \subsection{Getting Started with Scrum@Scale}
 When implementing large networks of teams, it is critical to develop a
 scalable \textbf{Reference Model} for a small set of teams. Any
 deficiencies in a Scrum implementation will be magnified when multiple
-teams are deployed.
+teams are deployed. Many of the initial scaling problems will be organizational 
+policies and procedures or development practices that block high performance and frustrate 
+employees. 
 
 Therefore, the first challenge is to create a small set of teams that
 implements Scrum well. This set of teams works through organizational
 issues that block agility and creates a Reference Model for Scrum that is
 known to work in the organization and can be used as a pattern for scaling
 Scrum across the organization.
 
-As the Reference Model of teams accelerates, impediments and bottlenecks
+As the Reference Model of teams is deployed in rolling waves across an organization, 
+impediments and bottlenecks
```

Comment:
this seems like a change to endorse a specific way of scaling the reference model that isn't what everyone does, we like the old section better